reference/orders.v2.oas2.yml

```
@@ -283,6 +306,38 @@ paths:
             schema:
               $ref: '#/components/schemas/order_Post'
             examples:
+              Order with Consignments:
+                value:
+                  status_id: 0
+                  customer_id: 12
+                  billing_address: []
```

This is an invalid value for billing address
```suggestion
                  billing_address:
                    first_name: Jane
                    last_name: Doe
                    street_1: 123 Main Street
                    city: Austin
                    state: Texas
                    zip: '78751'
                    country: United States
                    country_iso2: US
                    email: janedoe@example.com
```